piccolo_admin/endpoints.py

```diff
@@ -612,6 +670,8 @@ def create_admin(
     rate_limit_provider: t.Optional[RateLimitProvider] = None,
     production: bool = False,
     site_name: str = "Piccolo Admin",
+    default_language_code: str = "auto",
```

Comment:
It should revert to `default_language_code: str = "en"` because it is the English is default language